spec/sentryIntegrationSpec.coffee

```diff
@@ -0,0 +1,109 @@
+Helper = require('hubot-test-helper')
+# helper loads all scripts passed a directory
+helper = new Helper('../scripts/sentry-notifications.coffee')
+
+http = require('http')
+nock = require('nock');
+
+
+process.env.EXPRESS_PORT = 12345
+
+mockSlackHangoutNotify = (channel, text) ->
+  nock('https://slack.com', allowUnmocked: false)
+    .post('/api/chat.postMessage',
+      channel: channel,
+      text: text,
+      username: 'Sentry Bot'
+      icon_url: '',
+      parse: 'full',
+      token: process.env.SLACK_API_TOKEN)
+    .reply(200, {
+      ok: false,
+      error: 'not_authed'
+    })
+
+makeRequest = (body, done) ->
```

Comment:
@tansaku @joaopapereira it appears we don't need this function in this particular file. i confirmed by commenting it out and the specs still pass.